includes/class-wc-google-analytics.php

```diff
@@ -401,6 +401,12 @@ protected function get_ecommerce_tracking_code( $order_id ) {
 			return '';
 		}
 
+		// Check order key.
+		$order_key = empty( $_GET['key'] ) ? '' : wc_clean( wp_unslash( $_GET['key'] ) );
```

Comment:
Do we need to ignore the phpcs rule `WordPress.Security.NonceVerification.Recommended`
In this case the key would be the nonce.